recipes/smalt/meta.yaml

```diff
@@ -0,0 +1,32 @@
+build:
+  number: 0
+
+package:
+  name: smalt
+  version: "0.7.6"
+source:
+  fn: smalt-0.7.6-static.tar.gz
+  url: https://sourceforge.net/projects/smalt/files/smalt-0.7.6-static.tar.gz/download
+  md5: 57ed011836fda8fb5e34604dd1858b5e
+requirements:
+  build:
+    - gcc
+    - autoconf
+    - automake
+    - libtool
+    - pkg-config
+    - python
```

Comment:
python 2 is used for testing ("make check") during the build.
this fails on osx so for now i've disabled building there.
i've removed python from runtime dependencies.